gregorypoole.txt

```diff
@@ -0,0 +1,127 @@
+https://codepen.io/gregory-poole/pen/OJPwpdV?editors=1100
+
+
+1. What is Semantic HTML? 
+A way to organize html to make it make sense. 
+
+    2. What is HTML used for? 
+        html is used to create web pages
+
+    3. What is an attribute and where do we put it? 
+        a attribute can put a link inside of a tag.
+
+    4. What is the h1 tag used for? How many times should I use it on a page?
+    A h1 tag is used for a heading 
+
+    5. Name two tags that have required attributes
+    href and img src 
+
+    6. What do we put in the head of our HTML document? 
+    heading
+
+    7. What is an id? 
+    an id creates a selector for CSS
+
+    8. What elements can I add an id to? 
+        any tags 
+
+    9. How many times can I use the same id on a page? 
+     you can only use a id once 
+
+    10. What is a class? 
+    a class is a descriptor for more than one tag
+
+    11. What elements can I add a class to? 
+    p, div, h1, h2, etc..
+
+    12. How many times can I use the same class on a page? 
+    as many as you want 
+    \
+    13. How do I get my link to open in a new tab?
+    add a blank to your a tag 
+
+    14. What is the alt attribute used for? 
+    to add text 
+
+    15. How do I reference an id?
+    #
+
+    16. What is the difference between a section and a div
+    they are just different ways to organize your code
+
+    17. What is CSS used for? 
+    styling 
+
+    18. How to we select an element? Example - every h2 on the page
+    put h2 {
+
+    }
+
+    19. What is the difference between a class and an id? - Give me an example of when I might use each one
+    a class can be a selector for more than one element an id is just for one element specificly. Could use a class to group together mutiple paragraphs and style them. For and id you would use it for styling one element.
+
+    20. How do we select classes in CSS?
+    use a period then the name of the class
+
+    21. How do we select a p element with a single class of “human””?
+    .human
+
+    22. What is a parent child selector? When would this be useful? 
+    selects all elements that are a direct child of the specified element.
+
+    23. How do you select all links within a div with the class of sidebar?
+        div.sidebar
+
```

Comment:
`div.sidebar` would select the div.. how would you select the links inside as well?